src/pages/api/register/create_account.ts

```diff
@@ -124,12 +140,21 @@ export default async function handler(
       passwordSalt,
       passwordHash,
       authPublicKey,
-      twitter,
-      telegram,
+      twitter: parsedTwitter,
+      telegram: parsedTelegram,
       bio,
     },
   });
 
+  // Create chip key
+  await prisma.chipKey.create({
```

Comment:
I thought we would make the signing keys before hand so we can pre-assign groups?

Reply:
we can keep this in for now to quickly test and iterate but yeah i think i misunderstood

Reply:
this is for testing, see: https://github.com/cursive-team/zk-summit/issues/28